annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -649,8 +649,8 @@ private String getVariantClassificationFromMap(String variant) {
             variantMap.put("inframe_insertion",             "In_Frame_Ins");
             variantMap.put("inframe_deletion",              "In_Frame_Del");    
             variantMap.put("missense_variant",              "Missense_Mutation");
-            variantMap.put("protein_altering_variant",      "Missense_Mutation"); // Not sure if this is correct
-            variantMap.put("coding_sequence_variant",       "Missense_Mutation");
+            variantMap.put("protein_altering_variant",      "Missense_Mutation"); // Not correct, code below to handle it
```

Comment:
Should we change the comment to "Not always correct, code below to handle exceptions"?